arangosh/Import/ImportHelper.cpp

```diff
@@ -175,8 +178,14 @@ ImportHelper::ImportHelper(ClientFeature const* client,
     }));
     _senderThreads.back()->start();
   }
- 
-  // wait until all sender threads are ready 
+
+  // should self tuning code activate?
+  if (_autoUploadSize) {
+    _autoTuneThread.reset(new AutoTuneThread(*this));
+    _autoTuneThread->start();
```

Comment:
do we need to shutdown the `_autoTuneThread` somewhere? I see a call to `start()`, but no call to `beginShutdown()`.
I think it will stop automatically when class `ImportHelper` gets destroyed, but do we need to stop it before to make it shut down earlier etc.?

Reply:
Wait ... _autoTuneThread is in a smart pointer.  So its destructor is called ... which calls shutdown() then pops the condition.  How much faster do you want it going down?

Reply:
I was wondering if we need to destroy or at least notify `_autoTuneThread` somewhere in `beginShutdown()` or in `stop()` of the ImportFeature.
Maybe we don't. I only wanted to exclude the possibility that we have started that thread and it hangs for 10 seconds somewhere in its condition and is not getting signalled because of some hard failure in the import. Then the thread would need up to 10 seconds until it notices that `isStopping()` has changed to true.